### DISCUSSION TITLE
empty folder rmdir

### DISCUSSION THREAD

**Discussion by simonmcnair** [RsyncProject/rsync#596] [Ideas]
I'd love it if there was an option to rmdir any empty folders after --remove-source-files
This is a niggling feature but one that I think would be popular and nice.  Most of the time people recommend running find -type d -empty -delete afterwards but you need to pipe in the folder name which is a hassle.

Love your work on rsync and am a big fan of Samba.  Thanks for all the work you do, you're stars.
Thanks :-)

**Comment by Semnodime** [comment]
This feature request has been suggested via https://github.com/RsyncProject/rsync/issues/276.
